components/FullwidthSlider/src/index.scss

```diff
@@ -379,13 +379,14 @@
 
   .denhaag-fullwidth-slider__pagination-bullet {
     position: var(--denhaag-fullwidth-slider-pagination-bullet-position);
-    background-color: var(--denhaag-fullwidth-slider-pagination-bullet-background-color);
     opacity: var(--denhaag-fullwidth-slider-pagination-bullet-opacity);
     width: var(--denhaag-fullwidth-slider-pagination-bullet-width);
     height: var(--denhaag-fullwidth-slider-pagination-bullet-height);
     border-radius: var(--denhaag-fullwidth-slider-pagination-bullet-border-radius);
     margin-inline-start: var(--denhaag-fullwidth-slider-pagination-bullet-margin-inline);
     margin-inline-end: var(--denhaag-fullwidth-slider-pagination-bullet-margin-inline);
+    border: var(--denhaag-fullwidth-slider-pagination-bullet-border);
```

Comment:
border en border-color voelt erg dubbel. Waarom is border-color geen onderdeel van de border shorthand?